lib/EasyPost/Service/BaseService.php

```diff
@@ -164,6 +165,10 @@ protected function getNextPageResources($class, $collection, $pageSize = null)
             'before_id' => $collectionArray[count($collectionArray) - 1]['id']
         ];
 
+        if ($optionalParams != null) {
```

Comment:
A more PHP appropriate syntax would be this:

```suggestion
        if (isset($optionalParams)) {
```